packages/dlp/test/gapic_v2beta1.js

```diff
@@ -0,0 +1,341 @@
+/*
```

Comment:
Not sure if this is related to what you're looking for, but the file in the repo for video-intelligence is hyphenated: https://github.com/GoogleCloudPlatform/google-cloud-node/blob/master/packages/video-intelligence/test/gapic-v1beta1.js